### ISSUE TITLE
Drawable.createFromPath generates OutOfMemory

### ISSUE THREAD

**Issue by vasiledoe** [mikepenz/MaterialDrawer#1255]
I'd like to use images from resources in your drawer, so I have:

`        PrimaryDrawerItem primaryDrawerItem = new PrimaryDrawerItem();
         primaryDrawerItem.withName("some name");
         primaryDrawerItem.withIcon(Drawable.createFromPath("path to img"));
`

this approach is working but in my ~10 element with icon drawer, I get a considerable heap increase each clean+recreate iteration like:

![hitech - cusersvasildesktopandroidhitech - app - appsrcmainjavacomhitech_moldova_newsuidrawerdrawermenucustomizer java - an](https://cloud.githubusercontent.com/assets/15106474/15139209/5df9d988-169d-11e6-84f7-d0d130c57595.jpg)

How can I manage that? I was trying to use `AbstractDrawerImageLoader` for images like:
```
DrawerImageLoader.init(new AbstractDrawerImageLoader() {
            @Override
            public void set(ImageView imageView, Uri uri, Drawable placeholder) {
                File file = new File(String.valueOf(uri));
                Picasso.with(mAppContext).load(file).placeholder(placeholder).into(imageView);

            }

            @Override
            public void cancel(ImageView imageView) {
                MyLogs.LOG("DrawerMenuCustomizer", "drawItems", "DrawerImageLoader - cancel");
                Picasso.with(imageView.getContext()).cancelRequest(imageView);
                            }
        });

```
But I'can't set `primaryDrawerItem.withIcon("path to my file");` because `withIcon()` don't have a implementation for String param

**Comment by mikepenz** [comment]
@vasiledoe the normal `PrimaryDrawerItem` does not allow you to do this. but you can implement a **CustomDrawerItem** which takes also a `String` as Icon. 

The sample app contains such a sample `CustomDrawerItem` doing a pretty similar thing. 
https://github.com/mikepenz/MaterialDrawer/blob/develop/app/src/main/java/com/mikepenz/materialdrawer/app/drawerItems/CustomUrlPrimaryDrawerItem.java

**Comment by mikepenz** [comment]
@vandyshev please report back. will close for now

**Comment by inteist** [comment]
I have commented on another similar issue but it seems that this issue may be more relevant. 

I have confirmed that re-creation of the drawer is the culprit (if I statically setup it in one activity then no crashes occur)

I still do not understand how specific my case is but I am not doing anything too particular and I would think that the lib should readily support the usage of vector drawables for the menu (list) icons.

Should I open a new issue with all this info combined?
